test/clangd_tests.vim

```diff
@@ -1625,7 +1625,7 @@ def g:Test_LspDiagsSubcmd()
   new XLspDiagsSubCmd.raku
 
   feedkeys(":LspDiag \<C-A>\<CR>", 'xt')
-  assert_equal('LspDiag first current here highlight last next prev show', @:)
+  assert_equal('LspDiag first current here highlight last next nextWrap prev prevWrap show', @:)
   feedkeys(":LspDiag highlight \<C-A>\<CR>", 'xt')
```

Comment:
Can you add new tests for the nextWrap and prevWrap functionality?

Reply:
I've added some commands and assertions to the existing `Test_LspDiag_Multi()` test, is that OK or do you want separate test functions?